tests/test_ostinato.py

```diff
@@ -0,0 +1,53 @@
+import numpy as np
+import numpy.testing as npt
+import stumpy
+from stumpy.ostinato import _get_central_motif
+import pytest
+
+
+def naive_consensus_search(tss, m):
+    """
+    Brute force consensus motif from
+    <https://www.cs.ucr.edu/~eamonn/consensus_Motif_ICDM_Long_version.pdf>
+
+    See Table 1
+
+    Note that there is a bug in the pseudocode at line 8 where `i` should be `j`.
+    This implementation fixes it.
+    """
+    k = len(tss)
+
+    rad = np.inf
+    ts_ind = 0
+    ss_ind = 0
+
+    for j in range(k):
+        radii = np.zeros(len(tss[j]) - m + 1)
+        for i in range(k):
+            if i != j:
+                mp = stumpy.stump(tss[j], m, tss[i], ignore_trivial=False)
+                radii = np.max((radii, mp[:, 0]), axis=0)
+        min_rad_index = np.argmin(radii)
+        min_rad = radii[min_rad_index]
+        if min_rad < rad:
+            rad = min_rad
+            ts_ind = j
+            ss_ind = min_rad_index
+
+    return _get_central_motif(tss, rad, ts_ind, ss_ind, m)
```

Comment:
Instead of using `_get_central_motif` is it possible to  create a `naive_get_central_motif` instead? And then, we should test `_get_central_motif` against `naive_get_central_motif`. I think it is important to have this test since it is so core to `ostinato`

Reply:
I don't fully understand this. Can you elaborate on this idea? How would a `naive_get_central_motif` work? I feel like my implementation of `_get_central_motif` is already quite naive :P

Reply:
Sorry for the initial vagueness and I agree that `ostinato._get_central_motif` is already quite simple but here is my concern. What happens when somebody incorrectly modifies or changes `ostinato._get_central_motif`? Since this test calls `ostinato._get_central_motif` then we have no way of verifying the "correctness" of `ostinato._get_central_motif`. So, if somebody decides to alter `ostinato._get_central_motif` to not compute anything and to always return the set of values `1, 2, 3` (for `rad, ts_ind, ss_ind`) then this test will always pass since the test is calling `ostinato._get_central_motif`. In other words, to be properly covered, we should have a separate test to verify the outputs of `ostinato._get_central_motif`.  Does that make sense?

Reply:
Sometimes, when the functions are already simple, I create a "naive" version by replacing all of the major `NumPy` calls with a more verbose/explicit `for-loop` (i.e., replace `np.argmin`, `np.isclose`, `np.max`, etc).  I know that I am being rather stubborn here and I can handle this afterward in #285

Reply:
Yes that makes sense. I'll give it a shot.